MultisigIntegrationTests/CoreData/NetworkTests.swift

```diff
@@ -42,22 +42,22 @@ class NetworkTests: CoreDataTestCase {
     }
 
     func test_exists() {
-        XCTAssertFalse(try Network.exists(1))
+        XCTAssertFalse(try Network.exists("1"))
         _ = try? makeNetwork(id: 1)
 
-        XCTAssertFalse(try Network.exists(2))
-        XCTAssertTrue(try Network.exists(1))
+        XCTAssertFalse(try Network.exists("2"))
+        XCTAssertTrue(try Network.exists("1"))
     }
 
     func test_by() {
-        var network = Network.by(1)
+        var network = Network.by("1")
         XCTAssertNil(network)
         _ = try? makeNetwork(id: 1)
 
-        network = Network.by(2)
+        network = Network.by("2")
```

Comment:
just a thought, the better design decision would be to model the chain id type as a separate type, ChainID, and that would have less changes throughout the code base.

Nothing to do because of this comment now :)